src/aleph/web/controllers/utils.py

```diff
@@ -66,6 +68,21 @@ def iter_pages(self, left_edge=2, left_current=2, right_current=5, right_edge=2)
                 last = num
 
 
+def make_date_filters(start: float, end: float, filter_key: str):
```

Comment:
Would it make sense to create a `NewType` named `TimeStamp` to specify that these floats represent timestamps ?

```suggestion
Timestamp = NewType('Timestamp', float)

def make_date_filters(start: Timestamp, end: Timestamp, filter_key: str):
```

Reply:
I have it in a corner of my head, but that's for after the big refactoring.

Reply:
OK